frontend_tests/node_tests/settings_user_groups.js

```diff
@@ -501,6 +478,37 @@ run_test('on_events', () => {
         return true;
     };
 
+    (function pencil_icon_triggered() {
```

Comment:
sorry for `pencil_icon_triggered` this is not even needed, I wrote it in very beginning will remove it in the next amend.